main.go

```diff
@@ -149,7 +150,12 @@ func handle(w http.ResponseWriter, r *http.Request) {
 			// This is to support path@version syntax.
 			// A combination of GOBIN and `go install` would not work due to:
 			// go: cannot install cross-compiled binaries when GOBIN is set
-			args := []string{"run", "-exec", "cp"}
+			exc, err := os.Executable()
+			if err != nil {
+				// attempt to invoke by common name
+				exc = "wasmserve"
```

Comment:
When is this error happen?

Reply:
There are a lot of implementations, but on Unix a readlink can fail, and on Windows I think it invokes a syscall or something, so I thought I'd add a fallback just in case.

Reply:
In this case we should not continue this process, right?

Reply:
fair enough. replace with a panic or log.Fatal?

Reply:
actually, http.Error, my mistake

Reply:
```go
http.Error(w, string(out), http.StatusInternalServerError)
return
```

is fine.